### PR TITLE
Link to system_requirements for exact PHP versions

### PR DESCRIPTION
**Pull request by mejo-**
One place less that needs constant updating.

Signed-off-by: Jonas <jonas@nextcloud.com>

### CODE REVIEW

File: admin_manual/installation/source_installation.rst [hunk 1]
```diff
@@ -33,7 +33,7 @@ If you get a result, the module is present.
 
 Required:
 
-* PHP 7.3, 7.4 or **8.0** (*recommended*)
+* PHP (see :doc:`system_requrements` for a list of supported versions)
```
Comment (blizzz) [inline admin_manual/installation/source_installation.rst:36]: can i keep the typo?

[the remark this answers is not in the document]
Comment (come-nc) [reply admin_manual/installation/source_installation.rst:36]:
```suggestion
* PHP (see :doc:`./system_requirements.rst` for a list of supported versions)
```

Comment (mejo-) [reply admin_manual/installation/source_installation.rst]: :gift_heart:

**Review by come-nc — APPROVED** [review]

File: admin_manual/installation/source_installation.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (mejo-) [reply admin_manual/installation/source_installation.rst]: Thanks a lot!

**Pull request merged**